lib/utils.js

```diff
@@ -43,19 +49,24 @@ function normalizeTemporals(params) {
     type.ms_per_interval = type.per_interval / type.interval;
     type.drip_interval = type.interval / type.per_interval;
   }
-
-  if (params.elevated_limits) {
-    type.elevated_limits = normalizeElevatedTemporals(params.elevated_limits);
-  }
-
-  return type;
+  return type
 }
 
-function normalizeElevatedTemporals(params) {
-  let type = normalizeTemporals(params);
+/**
+ *
+ * @param {type} params
```

Comment:
Nit: great opportunity to rename `{type}` 😅

Reply:
yea, very good suggestion. Maybe `Bucket`?